### ISSUE TITLE
sql error setting prefix (Incorrect string value)

### ISSUE THREAD

**Issue by PhanaticD** [lucko/LuckPerms#2216]
error: https://pastebin.com/raw/en7vQzNJ

lp info:
```
[12:27:05 INFO]: [LP] Running LuckPerms v5.0.72 by Luck.
[12:27:05 INFO]: [LP] -  Platform: Bukkit
[12:27:05 INFO]: [LP] -  Server Brand: Paper
[12:27:05 INFO]: [LP] -  Server Version:
[12:27:05 INFO]: [LP] -  git-Paper-1595 (MC: 1.12.2) - 1.12.2-R0.1-SNAPSHOT
[12:27:05 INFO]: [LP] -  Storage:
[12:27:05 INFO]: [LP] -     Type: MariaDB
[12:27:05 INFO]: [LP] -     Ping: 0ms
[12:27:05 INFO]: [LP] -     Connected: true
[12:27:05 INFO]: [LP] [LP] -  Extensions:
[12:27:05 INFO]: [LP] -     me.lucko.luckperms.extension.legacyapi.LegacyApiExtension
[12:27:05 INFO]: [LP] -  Messaging: Sql
[12:27:05 INFO]: [LP] -  Instance:
[12:27:05 INFO]: [LP] -     Static contexts: None
[12:27:05 INFO]: [LP] -     Online Players: 6 (265 unique)
[12:27:05 INFO]: [LP] -     Uptime: 7h 50m 49s
[12:27:05 INFO]: [LP] -     Local Data: 6 users, 136 groups, 0 tracks

```

**Comment by Tobi406** [comment]
See #1586

**Comment by PhanaticD** [comment]
I noticed all the other tables except the messenger table were utf8mb_general_ci

**Comment by lucko** [comment]
This was fixed in dc5c6b013826695ce842ac1ce418ae335765cb53 - but existing tables won't be updated.

You can either update the encoding yourself, or delete the table and have LP recreate it.
